src/ripple/app/paths/cursor/RippleLiquidity.cpp

```diff
@@ -195,7 +195,7 @@ void rippleLiquidity (
 }
 
 static
-std::uint32_t
+Rate
 rippleQuality (
```

Comment:
Since `rippleQuality()` now returns a `Rate` it should probably be renamed to `rippleRate()` at some time in the future.  But this too can wait for a later pull request.